Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IqtFitMultiple.py

```diff
@@ -64,6 +64,11 @@ def PyInit(self):
         self.declareProperty(name='ConstrainIntensities', defaultValue=False,
                              doc="If the Intensities should be constrained during the fit")
 
+        self.declareProperty(name='ExtractMembers', defaultValue=False,
+                             doc="If true, then each member of the fit will be extracted, into their"
+                                 "own workspace. These workspaces will have a histogram for each spectrum "
+                                 "(Q-value) and will be grouped.")
```

Comment:
I cannot read the rest of this tool-tip documentation. (Ubuntu 16.04).
Maybe too many `"`s ?